packages/api/models/nomination.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  
+  class Nomination extends Model {};
+
+  Nomination.init({
+    id: DataTypes.UUIDV4,
```

Comment:
In looking at the way Sequelize otherwise handles primary keys/id fields, I think we can delete this line.